src/server.js

```diff
@@ -91,6 +97,7 @@ function index(req, res, next) {
 			config
 		);
 		var template = _.template(file);
+		res.setHeader("Content-Security-Policy", "default-src *; style-src * 'unsafe-inline'; script-src 'self'; reflected-xss block");
```

Comment:
I really like this policy! A few minor comments/questions:

- `Content-Security-Policy` must be set with all requests, any reason why you put it here instead of inside your `allRequests` function?
- `*` is the default value for everything so we don't need to specify it for `default-src`. Was there a specific reason why you did so?
- I'm not sure I fully understand `reflected-xss block`: is it to enforce Chrome's filter against executing code that exists in the URL (for example) onto all browsers that would not enforce it by default? 
- It's possible that some or all of the following directives could be added, provided that I'm not entirely wrong:
  - Any reason against `connect-src 'self';` considering how the server connects to the client (at least currently)?
  - I think we could set `child-src 'none'; object-src 'none';` as well, as I don't think we need framed content or Flash content or etc.
  - We can set `form-action 'self';` as well, or maybe even `form-action 'none';` as none of the forms get actually submitted.
  - We might be able to set `frame-ancestors 'none';` but maybe that could restrict usage for some users? (Thinking of the integrations like Bytesized or Cozy or whatever, maybe someone does crazy stuff to display their apps?)
  - We can set `media-src 'self'` as we don't embed videos or similar (but we do load one sound from within the app)